src/main/resources/jira-to-github-users.properties

```diff
@@ -52,4 +52,14 @@
 # Spring Web Flow migration
 # ==========================
 
-rstoya05-aop:rstoyanchev
+#rstoya05-aop:rstoyanchev
+
+# ==========================
+# Spring Integration migration
+# ==========================
+
+abilan:artembilan
+mark.fisher:markfisher
+tmarshall:trevormarshall
+david_syer:dsyer
+grussell:garyrussell
```

Comment:
Make sure everyone is still a contributor and can be assigned to issues, or otherwise the import will fail for issues with an invalid assignee.

Reply:
Yes, that's exactly what I did before starting migration.
And this PR is really a result of full migration and JIRA archivation.